From
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=24688d64544b43f2c14be54531ad8764419dde09

Fix build error:

libtool: link: gcc -O2 -Wall -I/usr/include/libusb-1.0 -I./common -I/usr/include/libxml2 -o cnijlgmon3 cnijcomif.o cnijifusb.o cnijlgmon3.o cnijifnet.o keytext.o cnijifnet2.o cnijutil.o  -L../../com/libs_bin64 -ldl -lusb-1.0 -lcnnet2 -lxml2 -lcnbpcnclapicom2 -lcnbpnet30 -lcups
/usr/bin/ld: cnijifnet2.o:(.bss+0x0): multiple definition of `GET_PROTOCOL'; cnijlgmon3.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status

diff -Naur cnijfilter2-source-5.90-1.orig/lgmon3/src/cnijlgmon3.c cnijfilter2-source-5.90-1/lgmon3/src/cnijlgmon3.c
--- cnijfilter2-source-5.90-1.orig/lgmon3/src/cnijlgmon3.c	2019-07-23 03:07:22.000000000 +0200
+++ cnijfilter2-source-5.90-1/lgmon3/src/cnijlgmon3.c	2021-08-06 19:15:46.556546000 +0200
@@ -55,7 +55,7 @@
 int (*GET_STATUS)(char *, int, int *, int * , char *);
 int (*GET_STATUS2)(char *, int, char *, int *, int * , char *, char *);
 int (*GET_STATUS2_MAINTENANCE)(char *, int, char *, int *, int * , char *, char *);
-int (*GET_PROTOCOL)(char *, size_t);
+static int (*GET_PROTOCOL)(char *, size_t);
 
 
  int main(int argc, char *argv[])
